cypress-test-order.md

```diff
@@ -0,0 +1,66 @@
+# Cypress test order
+
+Cypress tests should be executed in the following order:
+
+1) Server signup test 
+  - 1.1 Create a user
+2) Server login test
+  - 2.1 Login tests
+3) All of admin tests, for creating, editing and deleting a user, template, role and application
+  - 3.1 Create a user
+  - 3.2 Create an application
+  - 3.3 Create a role
+  - 3.4 Create a template 
+  - 3.5 Edit a user
+  - 3.6 Edit an application
+  - 3.7 Edit a role
+  - 3.8 Edit a template
+  - 3.9 Delete a user
+  - 3.10 Delete an application
+  - 3.11 Delete a role
+  - 3.12 Delete a template
+4) Topbar admin tests
+  - 4.1 Helper dropdown button functionalities
+  - 4.2 Avatar button functionalities
+5) Compose tests for creating and editing a namespace, module, record page and a record
```

Comment:
Just 'Compose tests'. Also, write a comment above this line saying that all compose tests will be run excluding delete tests because we'll need the resources for future tests.